zebra-rpc/src/methods/tests/vectors.rs

```diff
@@ -113,7 +113,10 @@ async fn rpc_getblock() {
                     .transactions
                     .iter()
                     .map(|tx| tx.hash().encode_hex())
-                    .collect()
+                    .collect(),
+                height: block.coinbase_height().unwrap(),
```

Comment:
Optional: could we use an `expect` call instead,
```suggestion
                height: block.coinbase_height().expect("test block should have coinbase height"),
```